SampleShared/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.3",
```

Comment:
consider increasing the version number here and keep it match the sample scenarios.

Reply:
good catch; thanks!